### PR TITLE
Add Mixmax app

### PR DESCRIPTION
**Pull request by wearhere**
"In your pull request, please make it clear if you are involved with the project" —> I am an engineer with Mixmax.

### CODE REVIEW

File: _data/apps.yml [hunk 1]
```diff
@@ -1848,3 +1848,12 @@
...
   icon: "cycligent.png"
+-
+  name: "Mixmax"
+  description: "See every action on your emails in real-time. Compose anywhere."
+  website: "https://mixmax.com/download"
+  keywords:
+    - "email"
+    - "analytics"
+  icon: "mixmax.png"
+  featuredOnHomepage: true
```
Comment (jlord) [inline _data/apps.yml]: This line needs to be removed and is causing the test failure (we aren't updating the homepage apps at this time).

Comment (wearhere) [inline _data/apps.yml]: Hm where can the apps be viewed if not on the homepage?

Comment (jlord) [inline _data/apps.yml]: http://electron.atom.io/apps has all of the apps that have been addd (nearly 200 now) the front page just shows 20.

Comment (wearhere) [inline _data/apps.yml]: Ah ok thanks! Fixed up.

**Comment by jlord** [comment]
Woot, awesome! Thanks!

**Pull request merged**